medusa/server/web/config/handler.py

```diff
@@ -2,16 +2,7 @@
 
 """Base handler for Config pages."""
 
-from __future__ import unicode_literals
```

Comment:
`__future__` imports should usually stay. This one in particular.
Basically, they're the first steps of making Python 2 compatible with Python 3.
Sorry for not being more explicit...